e2e/cephfs.go

```diff
@@ -254,6 +254,12 @@ var _ = Describe("cephfs", func() {
 			snapshotPath := cephFSExamplePath + "snapshot.yaml"
 			appEphemeralPath := cephFSExamplePath + "pod-ephemeral.yaml"
 			pvcRWOPPath := cephFSExamplePath + "pvc-rwop.yaml"
+
+			metadataPool, err0 := getCephFSMetadataPoolName(f, fileSystemName)
```

Comment:
```suggestion
var metadataPool string
metadataPool, err = getCephFSMetadataPoolName(f, fileSystemName)
if err != nil {
	e2elog.Failf("failed getting cephFS metadata pool name: %v", err)
}
```

Reply:
This will lead to shadow declaration of `err` in everywhere!!

Reply:
have you tried calling this function in Context outside of It at line 245? it should work, can you please check

Reply:
No! let me give it a try.

Reply:
Done now

Reply:
This seem to panic:
https://jenkins-ceph-csi.apps.ocp.ci.centos.org/blue/organizations/jenkins/mini-e2e_k8s-1.23/detail/mini-e2e_k8s-1.23/1505/pipeline